test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -235,8 +235,8 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
             $observed | Should -BeExactly "h-llo"
         }
 
-        It "Empty command should fail" {
-            & $powershell -noprofile -c ''
+        It "Missing command should fail" {
```

Comment:
Should there be a new test for empty command?

Reply:
`& $powershell -noprofile -c ''` is now a valid command (it returns an empty string) which is similar to the the next test, but I can add a test case specifically for an empty string.